Source/Tools/LinuxEmulation/ArchHelpers/MContext.h

```diff
@@ -203,9 +203,12 @@ constexpr static uint64_t ESR1_DataAbort_Level_EL2 = 0b01;
 constexpr static uint64_t ESR1_DataAbort_Level_EL1 = 0b10;
 constexpr static uint64_t ESR1_DataAbort_Level_EL0 = 0b11;
 
+const char* GetESRName(uint64_t ESR);
```

Comment:
Perhaps this should be a string_view, since this only ever returns valid strings

Reply:
Switched over to string_view.